src/envoy/http/authn/context.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+package Envoy.Http.IstioAuthN;
+
+// Container to hold authenticated attributes from JWT.
+message JwtPayload {
+  string user = 1;
+  string issuer = 2;
+  string audience = 3;
+  string presenter = 4;
+  map<string, string> claims = 5;
+}
+
+// Container to hold authenticated attributes from X509 (mTLS).
+message X509Payload {
+  string user = 1;
+};
+
+message AuthenticatePayload {
+  oneof payload {
+    X509Payload x509 = 1;
+    JwtPayload jwt = 2;
+  }
+}
+message Context {
+  string principal = 1;
```

Comment:
All fields in the proto should have comments.

Reply:
Done